src/app/Router.tsx

```diff
@@ -15,22 +18,17 @@ function AppRouter() {
   return (
     <Router>
       <Routes>
+        <Route path="/" element={<HomePage />}></Route>
+        <Route path="/smartpackage" element={<SmartPackage />}></Route>
+        <Route path="/smartpackage/:id" element={<PackageDetails />}></Route>
```

Comment:
Nitpick: I left a message on slack but we need to add new routes at the bottom to avoid touching other lines of code